lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -192,7 +192,7 @@ public function getDropIndexSQL($index, $table = null)
 
         return "IF EXISTS (SELECT * FROM sysobjects WHERE name = '$index')
```

Comment:
What the heck about this `IF EXISTS` SQL oO wasn't aware that we have this in the platform... Why exactly is that necessary? Looks really hackish. My opinion is that if I call this method I already know that the index exists... Can you clear me up? =)